docs/manual/users_and_groups.md

```diff
@@ -25,6 +25,9 @@ user/group allocation altogether by using
 
 ## Dependencies
 
+Explict group membership (m) will create a dependency on both the user
+and the group name.
```

Comment:
It's a bit weird to have this as the first thing in this section. I'd put it after the more common main file ownership stuff.